### ISSUE TITLE
Image 6.3.1 on dockerhub delivers 6.3.0.1120 

### ISSUE THREAD

**Issue by noqqe** [gurock/testrail-docker#12]
Hi,

the testrail-apache docker image on https://hub.docker.com/r/testrail/apache/tags is containing the wrong version

```
# docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
testrail/apache     6.3.1               2f3921e47b48        7 days ago          745MB
testrail/apache     6.3.1.1004          2f3921e47b48        7 days ago          745MB
```

And in the Admin Interface its **6.3.0.1120** after install. 

See: 

https://hub.docker.com/layers/testrail/apache/6.3.1.1004/images/sha256-2d3ed4b2cddc3bc35d2b0d902f09da5366cc15850b07e16582b548a8168ee905?context=explore

**Comment by HlivSeTar** [comment]
Hi Florian,
we apologize for this problem. The problem is fixed and here is the updated tag [6.3.1.1006](https://hub.docker.com/layers/testrail/apache/6.3.1.1006/images/sha256-2d3ed4b2cddc3bc35d2b0d902f09da5366cc15850b07e16582b548a8168ee905?context=explore).

**Comment by noqqe** [comment]
Ok solved
